devmode/src/main/resources/reference.conf

```diff
@@ -0,0 +1,7 @@
+kalix.dev-mode {
+    enabled = true
```

Comment:
Enable by default for now, but we may consider it an opt-in in an initial phase. Users that are now used to use docker-compose will have a port conflict if we start testcontainers on their behalf.